src/main/java/ToDo.java

```diff
@@ -0,0 +1,12 @@
+public class Todo extends Task {
+    public Todo (String description){
+        super(description);
+    }
+
+    // Prints task
+    public void printTask(){
+        System.out.print("[T][");
+
+        System.out.println("] " + description);
+    }
+}
```

Comment:
Missing EOF. Add an extra empty line at the end of file (EOF), as this indicates the end of the file.

You may configure IntelliJ to add this automatically for you when hit `CTRL/CMD + S`.